src/platform.jl

```diff
@@ -60,33 +60,6 @@ function info(p::Platform, pinfo::CL_platform_info)
     return CLString(result)
 end
 
-
-let info_map = Dict{Symbol, CL_platform_info}(
```

Comment:
Why did you delete this?

Reply:
This function is a repetition of Line#9.